zk/tests/zktls/methods/guest/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "verity_zk_tests_zktls_guest"
+version = "0.1.0"
+edition = "2021"
+
+[workspace]
+
+[dependencies]
+risc0-zkvm = { version = "1.1.2", default-features = false, features = ['std'] }
+verity-local-verify = { path = "../../../../../rs/local-verify" }
+verity-verifier = { path = "../../../../../../verity/verifier", default-features = false, features = [
```

Comment:
@victorshevtsov - `verity-verifier` across the zkVM and zkTLS demos - not sure if it's referencing `verity` submodule.

Reply:
Yes, It definitely references the verity submodule.